core/src/test/scala/io/projectglow/bgen/BgenRowConverterSuite.scala

```diff
@@ -49,6 +50,7 @@ class BgenRowConverterSuite extends BgenConverterBaseTest {
     val vcfRows = vcfDf
       .queryExecution
       .toRdd
+      .map(_.copy())
```

Comment:
nit: can we just move the `converter.convert` here?

Reply:
I don't believe we can - `converter` is not serializable.

Reply:
How about `.mapPartitions { it => val converter =... }`?

Reply:
The copy is just pretty confusing